### PR TITLE
Update set-column.md

### PR DESCRIPTION
**Pull request by adinilfeld**
Minor updates to the markdown file for `set-column`.

### CODE REVIEW

File: wrangler-docs/directives/set-column.md [hunk 1]
```diff
@@ -62,6 +62,21 @@ would result in this record:
...
+Arithmetic operations can be used in several ways. 
+- To apply a simple arithmetic operation to a single, non-Decimal column, mathematical notation can be used, e.g. 
+```
+set-column :output wage1 * 2
+```
+- To apply an operation to a single column of type Decimal, use the `decimal` operations, e.g. 
+```
+set-column :decimal_op_1 decimal:add(wage_1,25)
+```
+- To apply operations to multiple columns (of any time), use the `arithmetic` operations, e.g.
```
Comment (ellliao) [inline wrangler-docs/directives/set-column.md]: Small typo: time -> type, otherwise LGTM!

Comment (adinilfeld) [reply wrangler-docs/directives/set-column.md]: Good catch!

**Review by ellliao — APPROVED** [review]
LGTM

File: wrangler-docs/directives/set-column.md [hunk 2]
```diff
@@ -62,6 +62,21 @@ would result in this record:
...
+```
+- To apply an operation to a single column of type Decimal, use the `decimal` operations, e.g. 
+```
+set-column :decimal_op_1 decimal:add(wage_1,25)
+```
+- To apply operations to multiple columns (of any type), use the `arithmetic` operations, e.g. 
+```
+set-column :arithmetic_op_1 arithmetic:add(wage_1,wage_2)
+```
+For more information on working with numbers in Wrangler, consult [this Wiki page](https://cdap.atlassian.net/wiki/spaces/DOCS/pages/413466692/Working+with+numbers).
```
Comment (rrielley76) [inline wrangler-docs/directives/set-column.md:78]: For Google style, can we say "For more information about working with numbers in Wrangler, see..."<<just add the text/link>>

File: wrangler-docs/directives/set-column.md [hunk 3]
```diff
@@ -62,6 +62,21 @@ would result in this record:
 
 ## Arithmetic and decimal operations example 
 
+Arithmetic operations can be used in several ways. 
+- To apply a simple arithmetic operation to a single, non-Decimal column, mathematical notation can be used, e.g.
```
Comment (rrielley76) [inline wrangler-docs/directives/set-column.md:66]: Can we change to active voice: "....use mathematical notation, for example:"

File: wrangler-docs/directives/set-column.md [hunk 4]
```diff
@@ -62,6 +62,21 @@ would result in this record:
 
 ## Arithmetic and decimal operations example 
 
+Arithmetic operations can be used in several ways. 
+- To apply a simple arithmetic operation to a single, non-Decimal column, mathematical notation can be used, e.g. 
+```
+set-column :output wage1 * 2
+```
+- To apply an operation to a single column of type Decimal, use the `decimal` operations, e.g.
```
Comment (rrielley76) [inline wrangler-docs/directives/set-column.md:70]: can you change e.g. to "for example:"

File: wrangler-docs/directives/set-column.md [hunk 5]
```diff
@@ -62,6 +62,21 @@ would result in this record:
...
+Arithmetic operations can be used in several ways. 
+- To apply a simple arithmetic operation to a single, non-Decimal column, mathematical notation can be used, e.g. 
+```
+set-column :output wage1 * 2
+```
+- To apply an operation to a single column of type Decimal, use the `decimal` operations, e.g. 
+```
+set-column :decimal_op_1 decimal:add(wage_1,25)
+```
+- To apply operations to multiple columns (of any type), use the `arithmetic` operations, e.g.
```
Comment (rrielley76) [inline wrangler-docs/directives/set-column.md:74]: Can you change e.g. to "for example:"

File: wrangler-docs/directives/set-column.md [hunk 6]
```diff
@@ -82,31 +97,31 @@ Using this record as an example:
...
-set-type wage_2 decimal
+set-column :arithmetic_op_1 arithmetic:add(wage_1,wage_2)
+set-column :arithmetic_op_2 arithmetic:minus(wage_4,wage_3)
+set-column :arithmetic_op_3 arithmetic:multiply(wage_3,wage_4)
+set-column :arithmetic_op_4 arithmetic:divideq(wage_4,wage_5)
+set-column :arithmetic_op_5 arithmetic:divider(wage_5,wage_6)
+set-column :arithmetic_op_6 arithmetic:lcm(wage_6,wage_7)
+set-column :arithmetic_op_7 arithmetic:equal(wage_7,wage_8)
+set-column :arithmetic_op_8 arithmetic:max(wage_7,wage_8)
+set-column :arithmetic_op_9 arithmetic:min(wage_7,wage_8)
```
Comment (rrielley76) [inline wrangler-docs/directives/set-column.md:108]: Do we need to include average too?

**Pull request merged**